src/lib.rs

```diff
@@ -1434,8 +1432,6 @@ impl_zeroable! {
     //
     // In this case we are allowed to use `T: ?Sized`, since all zeros is the `None` variant.
     {<T: ?Sized>} Option<NonNull<T>>,
-    #[cfg(any(feature = "std", feature = "alloc"))]
-    {<T: ?Sized>} Option<Box<T>>,
```

Comment:
Ditto here.

Reply:
Ditto: since the kernel doesn't really have this fix yet (because the fix didn't contain the userspace side when we did it in the kernel), it makes more sense to send this as a fix after this sync is merged, so resolving.